scripts/reindex.py

```diff
@@ -105,9 +109,20 @@ def main(argv: List[str]):
 
     if args.index:
         logger.info('Queuing notifications for reindexing ...')
+        if args.source:
+            if not args.partition_prefix_length:
+                parser.error('Cannot specify a source when performing a local reindex')
+            for catalog in args.catalogs:
+                require(args.source in azul.catalog_sources(catalog),
+                        f'Source {args.source!r} not found in catalog '
+                        f'{catalog!r}')
```

Comment:
```suggestion
                        f'Source {args.source!r} not found in catalog {catalog!r}')
```

Reply:
I think this count as a "message" and thus should be wrapped at 80 chars (absolute) according to the contributing guide : 

> For prose (documentation, messages, comments) wrap lines at the word boundary closest to or at, but not beyond, column 79. The first column is column 0.

Let me know if you think I'm misinterpreting this section.